src/components/controller/appLoader.ts

```diff
@@ -1,10 +1,11 @@
 import Loader from './loader';
+import { Options } from '../app/types';
 
 class AppLoader extends Loader {
     constructor() {
         super('https://newsapi.org/v2/', {
-            apiKey: '', // получите свой ключ https://newsapi.org/
-        });
+            apiKey: '7087e28f9f2e4a09b9c0b8d9592f3dd9',
```

Comment:
it is better to use enum or const instead string
```suggestion
            apiKey: API_KEY,
```
and add const API_KEY = '7087e28f9f2e4a09b9c0b8d9592f3dd9';